donut.css

```diff
@@ -0,0 +1,26 @@
+body {
```

Comment:
Good start with the CSS. Try playing around more with the responsiveness of a site, if you've not already. that would mean moving away from using px, and using rem or em instead. It would also include some percentage-based width or height assignments.